app/code/Magento/Authorization/Test/Unit/Model/CompositeUserContextTest.php

```diff
@@ -6,32 +6,36 @@
 
```

Comment:
```suggestion
declare(strict_types=1);

```

Reply:
Declaration has been added.

Reply:
Strict typing declaration has been added to each test script.